lib/internal/http2/compat.js

```diff
@@ -164,7 +172,16 @@ class Http2ServerRequest extends Readable {
   }
 
   get trailers() {
-    return this[kTrailers];
+    return this[kTrailers] || [];
```

Comment:
this is not the exact behavior. The returned `this[kTrailers]`  will be editable, while `[]` will not.

Reply:
Hmm, good point, thanks! Will update.